src/System.Collections.NonGeneric/src/System/Collections/ArrayList.cs

```diff
@@ -602,6 +775,11 @@ public virtual int LastIndexOf(Object value, int startIndex, int count)
 #if FEATURE_NETCORE
         [FriendAccessAllowed]
 #endif
+        /// <summary>Returns a read-only <see cref="T:System.Collections.IList" /> wrapper.</summary>
```

Comment:
Good point, we'll look into this.  Running it on other files it looks like the conditionally defined attributes are the only ones it inserts below.